### ISSUE TITLE
Strike old copies of verovio toolkit from repo history

### ISSUE THREAD

**Issue by ahwitz** [DDMAL/vida.js#24]

**Comment by ahankinson** [comment]
Are the binaries too big?

**Comment by ahwitz** [comment]
Yup. I don't think it's worth having three copies in each commit spread throughout this and the VerovioEditor repo when they're all 2-7MB each. It might also not be worth the surgery required to complete this task, but we'll see.

**Comment by ahankinson** [comment]
Pretty easy, if not so elegant (basically you're rewriting your commit history, copying old commits to a new commit).

**Comment by ahwitz** [comment]
I've done something similar before many times, the issue is that it may not be as simple as one `git filter-branch` since the version numbers change commit to commit.

**Comment by ahankinson** [comment]
Regex?

**Comment by ahwitz** [comment]
I want to make sure I'm keeping the correct copy in the repo for each commit, and I'm not sure how consistent my stupidity was. Once I have time to do it, we'll see - this is rather low on my priority list at the moment.
